beacon-chain/powchain/service.go

```diff
@@ -479,30 +456,8 @@ func (s *Service) closeClients() {
 	}
 }
 
-func (s *Service) waitForConnection() {
```

Comment:
This function had some complex logic around checking for eth1 nodes falling out of sync. However, we cannot use this behavior for the merge